backend/src/emissions/models.py

```diff
@@ -41,6 +42,12 @@ def __str__(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def random_username(sender, instance, **kwargs):
```

Comment:
do we need this? i'm not sure i understood what this is doing

Reply:
looks like it uses the email instead of username, this function should maybe be renamed. although, ideally, we should not have anything to do with username anymore and directly use email

Reply:
this is a change that @redfrexx merged in the backend... what was the intention of this function @redfrexx ?

Reply:
yes, it's a bit of a workaround solution which sets the email as a dummy username. At the moment our `CustomUser` model inherits from `django.contrib.auth.models.AbstractUser` which requires a username. So I couldn't get rid of it without rewriting the whole model.  I'll look at the user permissions (#23) today, so might be the case that the `CustomUser` model has to be adapted anyways, then I would remove the username along with it.

Reply:
ok so I've renamed the function and moved it to a new module `signals.py`, since there will be more functions like this to make sure that all tables stay consistent when they are updated (e.g. needed for #15). Also instead of the email the user id is now used as a dummy username. (see #163) I think this is the easiest solution for now, but we can also discuss this in the next meeting.